### PR TITLE
Update to v0.13rc1 

### PR DESCRIPTION
**Pull request by raybellwaves**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by raybellwaves** [comment]
Waiting for https://github.com/anitagraser/movingpandas/pull/260 and a release

File: recipe/meta.yaml [hunk 1]
```diff
@@ -19,17 +19,12 @@ requirements:
     - pip
   run:
     - python >=3.7
+    - geopandas
```
Comment (jorisvandenbossche) [inline recipe/meta.yaml]: You could also use `geopandas-base` here if you further want to limit the default set of packages (compared to `geopandas`, that would avoid fiona/gdal, mapclassify (which requires scikit-learn, networkx) and folium)

Comment (anitagraser) [reply recipe/meta.yaml]: Nice, let me try that.

**Comment by anitagraser** [comment]
@conda-forge-admin, please rerender

**Review by anitagraser — APPROVED** [review]

**Pull request merged**